CHANGES.md

```diff
@@ -1,5 +1,9 @@
 Change Log
 ==========
+### 1.36 - 2017-08-17
```

Comment:
This will be `1.37 - 2017-09-01`. It's not a breaking change, so no need to list it as one. Usually we say something like  `Fixed a bug where developer error was thrown in Sandcastle example [#5703](https://github.com/AnalyticalGraphicsInc/cesium/issues/5703)`